### PR TITLE
feat: make team name configuable in the report url

### PR DESCRIPTION
**Pull request by garethjevans**

### CODE REVIEW

File: pkg/foghorn/controller.go [hunk 1]
```diff
@@ -422,9 +429,14 @@ func (c *Controller) getReportURLBase() string {
 	return os.Getenv("LIGHTHOUSE_REPORT_URL_BASE")
 }
 
+// getReportURLTeam gets the team to construct the report url
+func (c *Controller) getReportURLTeam() string {
+	return os.Getenv("LIGHTHOUSE_REPORT_URL_TEAM")
```
Comment (mrageh) [inline pkg/foghorn/controller.go:434]: What happens if this env variable is not set?

Should we guard against that or will it always be present?

File: pkg/foghorn/controller.go [hunk 2]
```diff
@@ -384,6 +384,13 @@ func (c *Controller) reportStatus(ns string, activity *jxv1.PipelineActivity, jo
 	}
 	urlBase := c.getReportURLBase()
 	if urlBase != "" {
+		urlTeam := c.getReportURLTeam()
+		team := ns
+		// override with env var if set
```
Comment (mrageh) [inline pkg/foghorn/controller.go:389]: Why do we override with env var if it set?

Comment (abayer) [reply pkg/foghorn/controller.go:389]: Because the namespace is the proper default.

**Pull request merged**